### PR TITLE
Some corrections to German Cheat Sheet

### PR DESCRIPTION
**Pull request by to-ba**

### CODE REVIEW

**Comment by daxtheduck** [comment]
@to-ba Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by moollaza** [comment]
@to-ba thanks for submitting this! In the future, please be sure to provide a more specific PR title (no one else will know what "some corrections" is referring to, and it also always helps to provide a description of your changes, and reasoning.

**Comment by to-ba** [comment]
@moollaza What means »PR title«?

**Comment by ColasBroux** [comment]
@to-ba PR title means Pull Request title, it is the title you give to your pull request :-)
For example your PR title is "Some corrections to German Cheat Sheet"

File: share/goodie/cheat_sheets/json/german.json [hunk 1]
```diff
@@ -112,8 +112,8 @@
       ],
       "Basics" : [
          {
-            "val" : "Meine Name ist...",
-            "key" : "My name is..."
+            "val" : "Meine Name ist …",
```
Comment (seisfeld) [inline share/goodie/cheat_sheets/json/german.json]: nit: Remove the blank here.

File: share/goodie/cheat_sheets/json/german.json [hunk 2]
```diff
@@ -70,7 +70,7 @@
             "key" : "Can you repeat, please?"
          },
          {
-            "val" : "Wie sagt man \"X\" auf Deutsch?",
+            "val" : "Wie sagt man »X« auf Deutsch?",
             "key" : "How do you say \"X\" in German?"
```
Comment (seisfeld) [inline share/goodie/cheat_sheets/json/german.json]: Shouldn't we use the same for both X?

**Comment by to-ba** [comment]
@seisfeld do not remove the space. Without the space it is wrong in german (and other european languages).

**Comment by seisfeld** [comment]
@to-ba Ah okay, good to know. Was not aware of that. So it's with a space in german and without a space in english then?

**Comment by kaenganxt** [comment]
You could also add the following to your PR:
In [line 115](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/share/goodie/cheat_sheets/json/german.json#L115) it should be "Mein Name ist..." instead of "Meine Name ist..."

**Comment by seisfeld** [comment]
Very correct @kaenganxt. This is a typo as well. Missed it in my review the last time I worked on this one. Thanks for pointing out.

**Comment by to-ba** [comment]
@kaenganxt Please see my comment above.
»Meine Name ist …«

**Comment by to-ba** [comment]
http://www.bibliothek.germanistik.rwth-aachen.de/auslassungszeichen.html

**Comment by seisfeld** [comment]
You are correct with that @to-ba. Still it's "Mein Name" not "MeinE Name".

File: share/goodie/cheat_sheets/json/german.json (hunk 2 above)
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/german.json]: I'd prefer `»X«`in both the English and German for consistency

File: share/goodie/cheat_sheets/json/german.json (hunk 1 above)
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/german.json]: Given that the ellipsis is replacing a word and doesn't have following punctuation the space is actually correct, however the key should also be: `My name is …`

**Comment by mintsoft** [comment]
@to-ba I've just put some additional feedback against the diff :+1:

**Comment by mintsoft** [comment]
Hiya @to-ba I'm just checking in to see if you're happy with the feedback provided; any issues just ping me :+1:

**Comment by to-ba** [comment]
@mintsoft I am happy :-)

**Comment by moollaza** [comment]
@to-ba there's no need to close this? You can add more commits to the branch you used and when you push the commits to the remote branch, here on GitHub your PR will update automatically

**Pull request closed without merging**